package.json

```diff
@@ -1,19 +1,23 @@
 {
-  "name": "branch-name",
+  "name": "branch-name-checker",
   "private": true,
   "main": "index.js",
   "scripts": {
     "start": "node ./index.js",
```

Comment:
The Dockerfile will take care of compiling the library so the command remains the same. However we can take different ways. Use ts-node, continue compiling the library, use Deno, ...

Reply:
Yes, for now let's use `ts-node index.ts` that most probably is the simpler solution, in a future iteration we can see other alternatives.

I am suggesting this change because if someone tries to run `npm run start` with the current solution, it will get an error indicating this files does not exist.

Reply:
Changed.